casts/countryPicker/src/UICountryPicker/CountryPicker.tsx

```diff
@@ -90,7 +90,7 @@ function useCountriesSearch(
     const [searching, setSearching] = React.useState(false);
     const [countriesList, setCountriesList] = React.useState<CountriesArray>([]);
     const [filteredList, setFilteredList] = React.useState(countriesList);
-    const fuse = React.useMemo(() => new Fuse(filteredList, fuseOptions), [filteredList]);
+    const fuse = React.useMemo(() => new Fuse(countriesList, fuseOptions), [filteredList]);
```

Comment:
hm, are u sure? doesn't it make `filteredList` obsolete? 🤔

Reply:
You goddamn right

Reply:
https://github.com/tonlabs/UIKit/pull/1509/commits/0ba443c16033b1e8a37d9f9f73c6d3521e142b76